src/opera/test/pge/rtc_s1/test_rtc_s1_pge.py

```diff
@@ -0,0 +1,132 @@
+#!/usr/bin/env python3
+
+"""
+==================
+test_rtc_s1_pge.py
+==================
+
+Unit tests for the pge/rtc_s1/rtc_s1_pge.py module.
+"""
+
+import glob
+import json
+import os
+import re
+import shutil
+import tempfile
+import unittest
+from io import StringIO
+from os.path import abspath, join
+
+from pkg_resources import resource_filename
+
+from opera.pge import RunConfig
+from opera.pge.rtc_s1.rtc_s1_pge import RtcS1Executor
+from opera.util import PgeLogger
+
+
+class RtcS1PgeTestCase(unittest.TestCase):
+
+    starting_dir = None
+    working_dir = None
+    test_dir = None
+
+    @classmethod
+    def setUpClass(cls) -> None:
+        """Set up directories and files for testing"""
+        cls.starting_dir = abspath(os.curdir)
+        cls.test_dir = resource_filename(__name__, "")
+        cls.data_dir = join(cls.test_dir, os.pardir, os.pardir, "data")
+
+        os.chdir(cls.test_dir)
+
+    @classmethod
+    def tearDownClass(cls) -> None:
+        """At completion re-establish starting directory"""
+        os.chdir(cls.starting_dir)
+
+    def setUp(self) -> None:
+        """Use the temporary directory as the working directory"""
+        self.working_dir = tempfile.TemporaryDirectory(
+            prefix="test_rtc_s1_pge_", suffix="_temp", dir=os.curdir
+        )
+
+        # Create the input dir expected by the test RunConfig and add
+        # dummy input files with the names expected by the RunConfig
+        input_dir = join(self.working_dir.name, "rtc_s1_test/input_dir")
+        os.makedirs(input_dir, exist_ok=True)
+
+        os.system(f"touch {join(input_dir, 'SAFE.zip')}")
+
+        os.system(f"touch {join(input_dir, 'ORBIT.EOF')}")
+
+        os.system(f"touch {join(input_dir, 'dem.tif')}")
+
+        os.chdir(self.working_dir.name)
+
+    def tearDown(self) -> None:
+        """Return to test directory"""
+        os.chdir(self.test_dir)
```

Comment:
This looks like it is always "None".  Is this desired behavior?

Reply:
`test_dir` is initialized to `None` at line 32, but `setUpClass` (which is called before any calls to `setUp` or `tearDown`) sets it to the location of the test directory in our repository.